ctx_test.go

```diff
@@ -46,3 +47,66 @@ func TestCtxSessCacheSizeOption(t *testing.T) {
 		t.Error("SetSessionCacheSize() does not save anything to ctx")
 	}
 }
+
+func TestCtxSetCipherSuites(t *testing.T) {
+	ctx, _ := NewCtx()
+	err := ctx.SetCipherSuites("TLS_AES_128_GCM_SHA256")
+	if err != nil {
+		t.Error("SetCipherSuites() returned unexpected error")
+		return
+	}
+
+	err = ctx.SetCipherSuites("invalid")
+	if err == nil {
+		t.Error("SetCipherSuites() did not return expected error")
+		return
+	}
+
+	if !strings.Contains(err.Error(), "no cipher match") {
+		t.Error("SetCipherSuites() did not return expected error")
+	}
```

Comment:
Probably worth running these two cases as subtests (`t.Run("case name", func(t *testing.T) {})`) to increase clarity.